homework21/Product.java

```diff
@@ -0,0 +1,55 @@
+import java.time.LocalDateTime;
+
+public class Product {
+    private String typ;
+    private double price;
+    private boolean discount;
+    private LocalDateTime timestamp;
+
+    public Product(String typ, long price, boolean discount) {
```

Comment:
Do not forget to rename constructor parameters as well

Reply:
+++